src/node-red/hardware/revolutionary-pi.md

```diff
@@ -0,0 +1,125 @@
+   keywords: node-red, raspberry pi, Revolutionary Pi
+---
+
+# {{meta.title}}
+
+Revolution Pi is an industrial-grade computer based on the Raspberry Pi Compute Module, designed specifically for use in industrial automation environments, offering features like robust hardware, a real-time operating system, and support for various industrial protocols, allowing it to be used as an IIoT gateway to collect and send data from factory machines to the cloud.
+
+## Goal
+
+This documentation guides users through installing the FlowFuse Device Agent on a Revolution Pi. The agent enables running Node-RED on the edge device, managing it remotely via FlowFuse, and accessing [enterprise features](/product/features/) that are useful for organizations. Since the device does not come with Node.js preinstalled, this guide covers installing both Node.js and the FlowFuse Device Agent.
+
+## Prerequisites
+
+Before proceeding with the installation, ensure you have the following:
+
+- **Revolutionary Pi** – A functioning device with internet access.
+- **Sudo Privileges** – Administrator access to install required packages.
+- **Network Access** – Ensure outbound traffic is allowed on port 443 for connections to:
+  
+  - `app.flowfuse.com:443`
+  - `mqtt.flowfuse.cloud:443`
+  
+- **Firewall Rules** – The Device Agent does not install Node-RED at startup. When receiving a snapshot, it downloads the required Node-RED version. Allow access to:
+
+  - `https://registry.npmjs.com`
+
+## Getting Started
+
+This guide provides step-by-step instructions for setting up the FlowFuse Device Agent on a Revolutionary Pi. It begins with installing Node.js, a prerequisite for running the agent, followed by the installation and configuration of the FlowFuse Device Agent.
+
+### Installing Node.js
+
+Since Node.js is not preinstalled on the Revolutionary Pi, follow these steps to install it:
+
+#### Step 1: Update System Packages
+
+```bash
+sudo apt update && sudo apt upgrade -y
+```
+
+#### Step 2: Install Required Dependencies
+
+Before adding the NodeSource repository, ensure that curl is installed, as it is required to download the setup script.
+
+```bash
+sudo apt install curl -y
+```
+
+#### Step 3: Install the Latest LTS Version of Node.js
+
+```bash
+curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash -
+```
+
+#### Step 4: Verify Installation
+
+```bash
+node -v
+```
+
+After running the verification command, the installed Node.js version should be displayed in the terminal.
+
+> **Note:** Ensure that the installed version is **Node.js 18 or later**,as this is the recommended version for optimal compatibility with the device agent.
+
+### Installing the FlowFuse Device Agent
+
+Once Node.js is installed, proceed with the installation of the FlowFuse Device Agent.
+
+#### Step 1: Install the FlowFuse Device Agent
+
+```bash
+sudo npm install -g @flowfuse/device-agent
+```
+
```

Comment:
Doesn't this work? https://github.com/FlowFuse/device-agent/blob/main/service/raspbian-install-device-agent.sh